pkg/handlers/webhookrhobsreceiver_test.go

```diff
@@ -35,7 +35,6 @@ var _ = Describe("RHOBS Webhook Handlers", func() {
 		testMFN          oav1alpha1.ManagedFleetNotification
 		testMFNR         oav1alpha1.ManagedFleetNotificationRecord
 		mockStatusWriter *clientmocks.MockStatusWriter
-		serviceLog       *ServiceLog
```

Comment:
We could use the new `*slv1.LogEntry` type for this?

Reply:
Yes, `*ServiceLog` is just the new `*slv1.LogEntry` type.